.github/workflows/kickstart-tests.yml

```diff
@@ -56,6 +56,9 @@ jobs:
             echo "::set-output name=skip_tests::rhel-only"
           elif [ $(echo "$TARGET_BRANCH" | grep -E "f[[:digit:]]*-(devel|release)") ]; then
             echo "::set-output name=skip_tests::rhel-only"
+          elif [ "$TARGET_BRANCH" == "rhel-8" ]; then
+            echo "::set-output name=skip_tests::fedora-only"
```

Comment:
Maybe we could skip also "rhel-9-only", like in
https://github.com/rhinstaller/kickstart-tests/blob/b4445c885d6298b986b0f46c593900fbacf25ab4/containers/runner/scenario#L23

Reply:
Good point!